Frank/donation

frankarinze opened the pull request.

frankrnz committed:
Merge branch 'development' of https://github.com/techfleetworks/Beela-phase-3 into development

frankrnz committed:
Merge branch 'development' of https://github.com/techfleetworks/Beela-phase-3 into development

frankrnz committed:
update on stripe integration

frankrnz committed:
Merge branch 'development' of https://github.com/techfleetworks/Beela-phase-3 into development

frankrnz committed:
update on build issue

techfleetworks commented on pages/index.tsx:
```diff
@@ -4,7 +4,7 @@ import Popper from "@popperjs/core"
 import Head from "next/head"
 
 import Testimonials from "../components/sections/homepage/Testimonials"
-import { Hero } from "../components/sections/homepage/Hero"
```
notice that the Testimonials section is being imported using all lowercase Homepage, and the Hero section is now being imported with the first H capitalized.

techfleetworks commented:
a suggestion would be to clone this branch onto an outside branch and once its fixed you can merge it to your branch and then submit the PR request.
its really strange that you cant see the errors on your Terminal.
maybe you should try researching that. other temmembers are able to see the errors on their terminal too.

techfleetworks commented:
I changes the letter on my local copy of your repo to see if that alone did the trick, so that i could submit the change and make the PR, but it showed other issues related to TSX on other sections that im unfamiliar with. let me know if youneed me to send those screenshots.

frankrnz committed:
Merge branch 'development' of https://github.com/techfleetworks/Beela-phase-3 into frank/donation

Solit0123 committed:
folder capitalized

it was capital H, wheras it was references inside the pages as lowercase,exept forthe hero where it was throwing error, so i changed the folder. and also the hero component wasnt being exported as default.
rfc
